pkg/pipelines/bootstrap.go

```diff
@@ -1,196 +1,199 @@
 package pipelines
 
 import (
-    "errors"
-    "fmt"
-    "io"
-    "os"
-
-    corev1 "k8s.io/api/core/v1"
-    v1rbac "k8s.io/api/rbac/v1"
-
-    "github.com/mitchellh/go-homedir"
-    "github.com/openshift/odo/pkg/pipelines/eventlisteners"
-    "github.com/openshift/odo/pkg/pipelines/meta"
-    "github.com/openshift/odo/pkg/pipelines/routes"
-    "github.com/openshift/odo/pkg/pipelines/tasks"
-    "github.com/openshift/odo/pkg/pipelines/triggers"
-    "sigs.k8s.io/yaml"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	corev1 "k8s.io/api/core/v1"
+	v1rbac "k8s.io/api/rbac/v1"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/openshift/odo/pkg/pipelines/eventlisteners"
+	"github.com/openshift/odo/pkg/pipelines/meta"
+	"github.com/openshift/odo/pkg/pipelines/routes"
+	"github.com/openshift/odo/pkg/pipelines/tasks"
+	"github.com/openshift/odo/pkg/pipelines/triggers"
+	"sigs.k8s.io/yaml"
 )
 
 var (
-    dockerSecretName = "regcred"
-    saName           = "pipeline"
-    roleName         = "tekton-triggers-openshift-demo"
-    roleBindingName  = "tekton-triggers-openshift-binding"
-
-    // PolicyRules to be bound to service account
-    rules = []v1rbac.PolicyRule{
-        v1rbac.PolicyRule{
-            APIGroups: []string{"tekton.dev"},
-            Resources: []string{"eventlisteners", "triggerbindings", "triggertemplates", "tasks", "taskruns"},
-            Verbs:     []string{"get"},
-        },
-        v1rbac.PolicyRule{
-            APIGroups: []string{"tekton.dev"},
-            Resources: []string{"pipelineruns", "pipelineresources", "taskruns"},
-            Verbs:     []string{"create"},
-        },
-    }
+	dockerSecretName     = "regcred"
+	saName               = "pipeline"
+	roleName             = "tekton-triggers-openshift-demo"
+	roleBindingName      = "tekton-triggers-openshift-binding"
+	devRoleBindingName   = "demo-sa-admin-dev"
+	stageRoleBindingName = "demo-sa-admin-stage"
+	// PolicyRules to be bound to service account
```

Comment:
Needs better names.   "demo-sa" is gone.

Reply:
made the mentioned changes